app/lib/actions/katello/repository/filtered_index_content.rb

```diff
@@ -23,10 +23,15 @@ def run
             unit_ids = search_units(repo)
             ::Katello::Deb.import_all(unit_ids, repo)
           elsif repo.yum?
-            unit_ids = search_units(repo)
-            if input[:content_type] == 'srpm'
+            if input[:dependency] && input[:dependency][:content_unit_href]
```

Comment:
lets not use 'dependency' here since we're actually using data from it.   Soething like 'upload_task' for a name would be a lot more clear

Reply:
so to clarify, i'd rename what gets passed in as well